WashingBot.js

```diff
@@ -46,7 +46,7 @@ function setupUI(){
 
     txtConnection = app.CreateText("", 0.8, 0.2, "Multiline");
     lay.AddChild(txtConnection);
-    txtConnection.SetText("Waiting...");
+    txtConnection.SetText("Slack notifications disabled");
```

Comment:
I agree that it should say "integration" instead of "notifications", but I'm not quite sure I understand why you put this message here.
At this point you still don't know if the notifications have been disabled or not, maybe this message should be inside the setupSlackBot method and shown only if the slackToken was not provided.